master/reference/calicoctl/index.md

```diff
@@ -3,7 +3,16 @@ title: calicoctl user reference
 ---
 
 The command line tool, `calicoctl`, makes it easy to manage Calico network
-and security policy.
+and security policy.  
+
+It can be downloaded from the [releases page of the 
+calicoctl repository](https://github.com/projectcalico/calicoctl/releases/). 
```

Comment:
Maybe link all the way to `/latest`? https://github.com/projectcalico/calicoctl/releases/latest